witchslabyrinth/WitchsLabyrinthGame
# Fix Koi Pond colliders

## Before (collider blocking pier outlined in red)
![image](https://user-images.githubusercontent.com/23004127/82254367-b8674400-9920-11ea-9f54-f985f6b0595d.png)


## After
![image](https://user-images.githubusercontent.com/23004127/82254283-91a90d80-9920-11ea-9e8d-0289f17c27de.png)


Adjusted collider blocking pier steps; player can now walk onto pier.
Generally organized collider hierarchy / prefab overrides as well.

No scene changes made, only relevant prefabs were modified

----

### Testing

1. Ensure you can walk up the pier steps and around the pier comfortably
2. Ensure you can't walk into the pond easily
    - I know it's possible because of my shitty movement code, but I want it to be unlikely to happen
